packages/itwinui-react/tsconfig.test.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "./tsconfig.json",
-  // TODO: Test all the files, and not just the Table files or just the build files.
-  "include": ["src/**/*.test.*", "src/**/*.types-test.*"]
+  "compilerOptions": {
+    "jsx": "react-jsx",
```

Comment:
Yes. As mentioned in the PR description, I didn't want to revert all the test files to the old transform because these are not relevant for our users.